apps/financial-aid/backend/src/app/modules/municipality/municipality.controller.ts

```diff
@@ -42,6 +43,15 @@ export class MunicipalityController {
     return municipality
   }
 
+  @Post('')
+  @ApiCreatedResponse({
+    type: MunicipalityModel,
+    description: 'Creates a new municipality',
+  })
+  create(@Body() input: CreateMunicipalityDto): Promise<MunicipalityModel> {
```

Comment:
Should this return `Municipality` instead of the model?

Reply:
Are we sure, the whole file is returning the model isn't it rather in the api we want to return the `**Municipality** ` ? 🙃

Reply:
ahh this is the controller my bad 🙈